vta/src/runtime.cc

```diff
@@ -37,6 +37,7 @@
 #include <thread>
 #include <memory>
 #include <atomic>
+#include <map>
```

Comment:
I didn't see anywhere used this header file. Is this really needed?

Reply:
Good catch I'll remove it.